lib/ansible/modules/hostname.py

```diff
@@ -1001,6 +1001,12 @@ class PopHostname(Hostname):
     strategy_class = DebianStrategy
 
 
+class RockyLinuxHostname(Hostname):
```

Comment:
```suggestion
class RockyHostname(Hostname):
```

As I noted in the Rocky mattermost, we should probably name it like this for consistency.